src/main/java/pl/xkoem/GameBoard.java

```diff
@@ -1,21 +1,22 @@
 package pl.xkoem;
 
+import pl.xkoem.userinterface.UserInterface;
+
 import java.security.InvalidParameterException;
-import java.util.function.Consumer;
 
 public class GameBoard {
 
     private Symbol[] boardSymbols;
     private final int width;
     private final int height;
-    private final Consumer<String> userOutput;
+    private final UserInterface userInterface;
     private int newestPosition;
 
-    public GameBoard(GameConfiguration gameConfiguration, Consumer<String> userOutput) {
+    public GameBoard(GameConfiguration gameConfiguration, UserInterface userInterface) {
         int[] boardSize = gameConfiguration.getBoardSize();
         width = boardSize[0];
         height = boardSize[1];
-        this.userOutput = userOutput;
+        this.userInterface = userInterface;
```

Comment:
Usuń this w tym, albo dodaj this w powyższych

Reply:
Done